src/plumbing/map.cljx

```diff
@@ -14,7 +14,8 @@
   "Like select-keys, but asserts that all keys are present."
   [m ks]
   (doseq [k ks]
-    (assert (contains? m k)))
+    (or (contains? m k)
+        (schema/assert-iae false "Key %s not found in %s" k (mapv key m))))
```

Comment:
If there are more than one missing keys, it would also be nice to show them all.